Langs/0419.lang

```diff
@@ -9,6 +9,8 @@ Medium=8.5
 Small=8.2
 Name=Arial
 [Strings]
+Enable enhanced accessibility?=Включить, "Улучшенную видимость"?
```

Comment:
@Agc0de6 (lots of pings lately lol)
Do you approve this and below string?

Reply:
@micwoj92 just delete comma ok string 12 and that's all.